msteams-platform/toolkit/toolkit-v4/add-single-sign-on-vs.md

```diff
@@ -25,16 +25,16 @@ Teams provides SSO function for an app using the Teams Toolkit for Microsoft Vis
 
     :::image type="content" source="../../assets/images/teams-toolkit-v2/teams-toolkit-vs/vs-add-authentication-code.PNG" alt-text="Screenshot shows the add authentication code.":::
 
-Teams Toolkit helps you generate the authentication files in **TeamsFx-Auth** folder, including a manifest template file for Azure AD application and authentication redirect pages. Link the files to your Teams application by updating authentication configurations to ensure the SSO works for your application.
+Teams Toolkit helps you generate the authentication files in **TeamsFx-Auth** folder, including a app manifest (previously called Teams app manifest) template file for Azure AD application and authentication redirect pages. Link the files to your Teams application by updating authentication configurations to ensure the SSO works for your application.
```

Comment:
Remove `(previously called Teams app manifest)` from this sentence, as this is talking about Azure AD manifest.

Reply:
Updated.